src/lib/spinel/spinel.h

```diff
@@ -2370,6 +2374,12 @@ enum
      */
     SPINEL_PROP_NET_PSKC = SPINEL_PROP_NET__BEGIN + 11,
 
+    /// Instruct NCP to leave the current network gracefully
+    /** Format `b` - Write only
+     *
+     */
+    SPINEL_PROP_NET_LEAVE_GRACEFULLY = SPINEL_PROP_NET__BEGIN + 12,
```

Comment:
Should we also update the function `spinel_prop_key_to_cstr`?

Reply:
Updated.